index.html

```diff
@@ -539,6 +538,17 @@ <h3>
           indicated by the <a>active</a> attribute changes.
         </p>
       </section>
+      <section> <h3>The <dfn>WakeLockRequestOptions</dfn> dictionary</h3>
+        <pre class="idl">
+          dictionary WakeLockRequestOptions {
+            AbortSignal? signal;
+          };
+        </pre>
+        <p>
+          The <dfn>WakeLockRequestOptions.signal</dfn> property allows to abort
```

Comment:
Do you mean, "allows |signal| to abort"?

Reply:
@marcoscaceres do you have ideas on how to best word this?

Fetch just has:
> signal
> An AbortSignal to set request’s signal.

> readonly attribute AbortSignal signal;
> request . signal
> Returns the signal associated with request, which is an AbortSignal object indicating whether or not request has been aborted, and its abort event handler.

So they also seem to expose the signal - maybe we should too- thought I don't really understand the use case